recipes/so3/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.3.3":
+    url: "https://github.com/astro-informatics/so3/archive/refs/tags/1.3.3rc1.zip"
```

Comment:
I intend to do a full release before this PR is merged, once I know it all works.